bin/improver-spot-extract

```diff
@@ -84,6 +90,24 @@ def main():
         " in altitude to the spot site within the search radius defined when"
         " the neighbour cube was created. May be used with land_constraint.")
 
+    percentile_group = parser.add_argument_group(
+        title="Extract percentiles",
+        description="Extract particular percentiles from probabilistic "
+        "or realization inputs. If deterministic input is provided a warning "
+        "is raised and all leading dimensions are included in the returned "
+        "spot-data cube.")
+    percentile_group.add_argument(
+        "--extract_percentile", default=None, nargs='+', type=int,
```

Comment:
Would be tempted to call this `--extract_percentiles` and maybe show an example with a list (or specify space-separated list in help string).

Reply:
Changed.